### PR TITLE
[WC-1388]: enable e2e

### PR DESCRIPTION
**Pull request by rahmanunver**
### Description

Restore end-to-end testing for widgets. Right now only e2e enabled only for `badge-web`, so, other packages will be enabled soon.

### Pull request type

-   [x] No code changes (changes to documentation, CI, metadata, etc)
-   [ ] Dependency changes (any modification to dependencies in `package.json`)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Test related change (New E2E test, test automation, etc.)

### CODE REVIEW

File: automation/run-e2e/docker/m2ee.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+m2ee:
```
Comment (iobuhov) [inline automation/run-e2e/docker/m2ee.yml:1]: This file is just copied from `widget-resources`, you may skip ip

File: automation/run-e2e/docker/mxbuild.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+FROM mono:6.12
```
Comment (iobuhov) [inline automation/run-e2e/docker/mxbuild.Dockerfile:1]: This file is just copied from `widget-resources`, you may skip ip

File: automation/run-e2e/docker/mxruntime.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,46 @@
+FROM adoptopenjdk/openjdk11:jdk-11.0.3_7
```
Comment (iobuhov) [inline automation/run-e2e/docker/mxruntime.Dockerfile:1]: This file is just copied from `widget-resources`, you may skip it

File: automation/run-e2e/docker/runtime.sh [hunk 4]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```
Comment (iobuhov) [inline automation/run-e2e/docker/runtime.sh:1]: This file is just copied from `widget-resources`, you may skip ip

**Review by iobuhov — APPROVED** [review]

**Review by r0b1n — APPROVED** [review]

**Pull request merged**